### DISCUSSION TITLE
Combobox multi-select bug

### DISCUSSION THREAD

**Discussion by karoltonik** [twilio-labs/paste#3432] [Help: Figma] [answered]
I noticed that in the case of a multi select combobox, the width of the listbox doesn't expand fully relative to the input. This occurs because for multi select, the constraints are set to top & left. The issue can be resolved by setting the constraints to top & left + right, similar to how it's done for single select, where it works correctly

Combobox multi select:
<img width="633" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/482f2dfa-3da0-44d6-80a7-ed95472bb09b">

Combobox multi select constrains:
<img width="237" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/a13c18dd-363d-4f56-bee7-ba393e0c0dcb">

Combobox single select:
<img width="566" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/5721d603-f16a-40f1-8473-1c61fbce898d">

Combobox single select constrains:
<img width="235" alt="image" src="https://github.com/twilio-labs/paste/assets/141224835/cb73be76-133f-4580-a253-8e86e117a718">

[Figma reference](https://www.figma.com/file/oeUbF1qpn2FUsmXwiTsHmN/Segment-App%3A-Paste-Migration?type=design&node-id=1731%3A610352&mode=design&t=sxz8A0E4jtTVxu2B-1)

**Comment by serifluous** [comment, marked as answer]
Hey @karoltonik, we have a fix for this in our component library. I'm just waiting on Figma to resolve some publishing issues on our file. I'll reply here when the fix is published.

**Reply by serifluous** [reply]
The fix has just been published in our Figma library!
